index.html

```diff
@@ -2385,24 +2392,281 @@ <h2>
 
     </section>
 
+
     <section>
-        <h2>
-CBOR
-        </h2>
+      <h2>CBOR</h2>
+   <p>
+Like Javascript Object Notation (JSON) <a>[[RFC8259]]</a>, Concise Binary Object Representation (CBOR) <a>[[RFC7049]]</a> 
+defines a set of formatting rules for the portable representation of structured data. CBOR aims to be a more concise machine-readible language-independent data interchange format that is self-describing and has built-in semantics for interoperability.  With specific constraints, CBOR can support all JSON data types (including JSON-LD) for translation between the DID document model (described in Sections § 6. Data Model and § 3.3 DID Documents) and other core representations.  
+  </p>
+
+  <p>Concise Data Definition Language (CDDL) [[RFC 8610 ]] is a notational convention used to express Concise Binary Object Representation (CBOR), and by extension JSON Data Structures. The following notation expresses the DID Document model in CBOR representation with specific constaints for deterministic mappings between other core representations. </p>
+
+  <pre class="example nohighlight" title="DID Document data model for CBOR expressed in CDDL notation">
+    DID-document = {
+      ? @context : uri                     
+      id : did  
+      ? publicKey : [* publicKey ]
+      ? authentication :  [ *did // *publicKey // *tstr ]  
+      ? service : [ + service  ]
+      ? controller : did / [ *did ]    
+      ? created : time 
+      ? updated : time
+      proof :  any
+    }
+    
+    publicKey = {
+      id : did 
+      type : text
+      controller : uri 
+    }
+    
+    did = tstr .pcre "^did\\:(?<method-name>[a-z0-9]{2,})\\:(?<method-specific-id>[A-Za-z0-9\\.\\-\\:\\_]+)"
+    
+    did-url =  tstr .pcre "^did\\:(?<method-name>[a-z0-9]{2,})\\:(?<method-specific-id>[A-Za-z0-9\\.\\-\\:\\_]+)\\;(?<path>[A-Za-z0-9\\/)(?<query>\\?[a-z0-9\\=\\&])#(?<fragment>.+)"
+    
+    service = {
+      id : did-url 
+      type : text 
+      serviceEndpoint : uri 
+      ? description : text 
+      * tstr => any 
+    }
+   </pre>
```

Comment:
As @burnburn pointed out this is a working draft and by that nature the model is a work in progress to define the core data model among different serialization.  In CBOR, this is the most up-to-date  of it that I would like to contribute.  As we constrain the model to the core model this can be revised to be more accurate.